sys/auto_init/auto_init.c

```diff
@@ -90,6 +94,9 @@ void auto_init(void)
     config_load();
 #endif
 
+#ifdef MODULE_TINYMT32
+    genrand_init(0);
```

Comment:
Wouldn't this lead to having exactly the same random numbers generated on each board?